geode-core/src/main/java/org/apache/geode/management/internal/i18n/CliStrings.java

```diff
@@ -1924,9 +1924,7 @@
   public static final String REMOVE__MSG__CLEARED_ALL_CLEARS = "Cleared all keys in the region";
```

Comment:
I know this wasn't something introduced in this commit, but I feel that this naming was a typo and should be `REMOVE__MSG__CLEARED_ALL_KEYS `